app/style/main.less

```diff
@@ -80,6 +80,13 @@
   color: @error-text-color;
 }
 
+.right {
```

Comment:
I looked but didn't find these styles being used. In this actually being used somewhere?